src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -497,13 +497,10 @@ private bool IsUnixCommandAvailable(string command)
         internal static extern bool RemoveDirectory(string lpFileName);
 
         [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
-        public static extern IntPtr FindFirstFile(string lpFileName, out WIN32_FIND_DATA lpFindFileData);
+        public static extern SafeFindHandle FindFirstFile(string lpFileName, out WIN32_FIND_DATA lpFindFileData);
```

Comment:
Why is this P/Invoking at all rather than just using Directory.EnumerateFiles?

Reply:
Mainly because there seem to be no tests. Also, it appears they're pinvoking to work around long path limits. @JeremyKuhne I guess that's unnecessary now?